src/Controls/src/Core/SetterSpecificity.cs

```diff
@@ -15,6 +15,10 @@ namespace Microsoft.Maui.Controls
 
 		public static readonly SetterSpecificity DynamicResourceSetter = new SetterSpecificity(0, 0, 1, 0, 0, 0, 0, 0);
 
+		public static readonly SetterSpecificity ImplicitSetter = new(StyleImplicit, 0, 0, 0);
+		public static readonly SetterSpecificity LocalSetter = new(StyleLocal, 0, 0, 0);
+		public static readonly SetterSpecificity LocalClassSetter = new(StyleLocal, 0, 1, 0);
```

Comment:
I think the main question here is if @StephaneDelcroix likes consolidating these into fields, as he is more familiar with this code. It does seem like it cleans up the code to me.

Reply:
Won't the reference to this field make a copy anyway? There should be no perf difference between new vs copy... Or maybe there is. No need to execute instructions, just copy memory...

Reply:
There might be some runtime optimization, where it is faster to copy from a field (I guess?). I'm just judging from the benchmark results. This only helps performance a small amount, so the question is if it cleans up the code. If it does, seems worth it to me.